### PR TITLE
Add keymap for jis

### PR DESCRIPTION
**Pull request by w1mvy**
日本語キーボード用のいくつかのkeymapを追加しました

### CODE REVIEW

**Comment by ardecvz** [comment]
Hi, Taishi!

According to your changes in specs, you'd like to output JIS_* family key definitions without the KeyCode:: prefix. Are you sure about that?

I found [this](https://github.com/tekezo/Karabiner/blob/master/src/core/server/Resources/include/checkbox/languages/japanese.xml) in the official repo and it seems that you should use JIS_* with KeyCode:: (e.g., KeyCode::JIS_EISUU).

So you may simply write in your `~/.karabiner` something like:
~~~
item 'Japanese madness :)' do
  remap 'JIS_EISUU', to: '1'
end
~~~
and this should work.

In other words, Karabiner DSL supports all raw key definitions which are supported by Karabiner, not only listed in `key.rb` file (it mostly contains handy aliases).

**Comment by w1mvy** [comment]
Hi, @ardecvz.  I got it. So, close this pull request. Thank you for explanation.

**Comment by ardecvz** [comment]
Honestly, I wanted to add this memo to the readme before as I had been confused about it too.

Anyway, thanks for your interest for the project! If you find some time to work on the ideas listed in the roadmap (or maybe your own), I'll really appreciate it since I need them hardly but sadly I haven't got time currently.
